File: javascript/1910. Remove All Occurrences of a Substring.js
/**
 * @param {string} s
 * @param {string} part
 * @return {string}
 */
var removeOccurrences = function (s, part) {
  while (true) {
    let i = s.indexOf(part);
    if (i == -1) break;
    s = s.slice(0, i) + s.slice(i + part.length);
  }

  return s;
};

File: c#/1910. Remove All Occurrences of a Substring.cs
public class Solution {
    public string RemoveOccurrences(string s, string part) {
        while (true) {
            int i = s.IndexOf(part); 
            if (i == -1) break;
            s = s[0..i] + s[(i + part.Length)..];
        }

        return s;
    }
}